lib/context-menu-handlers.js

```diff
@@ -17,6 +18,23 @@ const contentScriptVimeoCase = `self.on('click', function (node, data) {
   self.postMessage('https://vimeo.com/' + node.href);
 });`
 
+const contextMenuMediaContentScript = `
+self.on('click', function (node, data) {
+  self.postMessage({
+    src: node.href || node.src,
+    url: window.location.href,
+    domain: new URL(node.href || node.src).host
```

Comment:
The `URL` constructor throws if the string isn't a valid URL. Could you wrap this in a try-catch, and not send the postmessage if it throws?